app/models/work_order_facade.rb

```diff
@@ -7,9 +7,9 @@ def initialize(reference)
     @reference = reference
   end
 
-  def related_properties
+  def related_facilities
```

Comment:
So are we now referring to estate, sub-block, lifts etc. all as facilities?

Reply:
For now, yes. I spoke to @Lawrencerichards and we decided to say _related facilities_ instead of _properties_ for now because not _all related properties_ are listed by this. (plus the API endpoint is _facilities_)

Reply:
Haha ok, it's just getting a bit confusing now. In the filter box we have on the right of the repairs history page they're referred to as the hierarchy.